iOS-28th-assignment/SignUpViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SignUpViewController.swift
+//  iOS-28th-assignment
+//
+//  Created by 한승현 on 1/28/24.
+//
+
+import UIKit
+
+class SignUpViewController: UIViewController {
+    // ID, Password, Password 확인 TextField 변수
+    @IBOutlet weak var idTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var passwordConfirmTextField: UITextField!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // UI 컴포넌트들을 초기화한다.
+        configureUiComponenets()
+    }
+    
+    // UI 컴포넌트를 configure하는 메서드
+    private func configureUiComponenets() {
+        configureIdTextField()
+        configurePasswordTextField()
+        configurePasswordConfirmTextField()
+    }
+    
+    // ID TextField를 configure하는 메서드
+    private func configureIdTextField() {
+        idTextField.placeholder = "이메일 또는 전화번호"
+        idTextField.keyboardType = .default
+        idTextField.clearButtonMode = .whileEditing
+    }
+    
+    // Password TextField를 configure하는 메서드
+    private func configurePasswordTextField() {
+        passwordTextField.placeholder = "비밀번호"
+        passwordTextField.isSecureTextEntry = true
+        passwordTextField.keyboardType = .default
+        passwordTextField.clearButtonMode = .whileEditing
+    }
+    
+    // Password 확인 TextField를 configure하는 메서드
+    private func configurePasswordConfirmTextField() {
+        passwordConfirmTextField.placeholder = "비밀번호 확인"
+        passwordConfirmTextField.isSecureTextEntry = true
+        passwordConfirmTextField.keyboardType = .default
+        passwordConfirmTextField.clearButtonMode = .whileEditing
+    }
+    
+    // ID, Password, Password 확인 입력을 검증하는 메서드
+    private func isValidInput() -> Bool {
+        // 유효조건 : ID, Password가 비어있지 않고, Password, Password 확인 값이 같음
+        return (idTextField.text?.isEmpty == false && passwordTextField.text?.isEmpty == false) && (passwordTextField.text == passwordConfirmTextField.text)
+    }
+    
+    // 회원가입 버튼 클릭 이벤트 핸들링 메서드
+    @IBAction func signUpButtonClicked(_ sender: Any) {
+        if (isValidInput()) {
```

Comment:
```suggestion
       guard isValidInput() else { return }
```
이런 식으로 유효하지 않은 경우 따로 처리해야 하는 로직이 없을 때 바로 return 되도록 할 수 있을 것 같네요 ~

Reply:
guard 이 친구는 수입후추에만 있어서 뭔가 아직 적응 안 됨 ... 일단 이런 것도 있다는 건 머리에 박고가요 ~